src/main/java/com/capstone/consumer/controllers/GetNoFlyZonesController.java

```diff
@@ -43,4 +44,21 @@ public GetNoFlyZonesResponse getNoFlyZones() {
         return serviceHandler.handle();
     }
 
+    
+
+    /**
+     * Sets up request mapping and request params for delete no-fly zone service
+     * Cross Origin scripting setup allows requests from any cross-origin script
+     *
+     * @param zoneName The name of the zone to be deleted
+     * @return String of deleted no-fly zone
+     */
+    @CrossOrigin(origins = "*")
+    @GetMapping("/getTfrNoFlyZones")
+    @ResponseBody
+    public GetNoFlyZonesResponse getTfrNoFlyZones() {
+        LOGGER.info("Received Request to get TFR no fly zones");
```

Comment:
Just a heads up. But we will likely remove all the info-level request logs and replace them by interceptors that automatically log the requests.